Стилизация страниц

Valeriia-2 opened the pull request:
---
:mortar_board: [Стилизация страниц](https://up.htmlacademy.ru/adaptive/25/user/1445103/tasks/6)

:boom: https://htmlacademy-adaptive.github.io/1445103-pink-25/4/

Valeriia-2 committed:
Update package-lock.json

Valeriia-2 committed:
ДЗ 3.8. Стилизация страниц

Подключила css на страницы

Valeriia-2 committed:
Добавила цвета (переменные), подключила блоки less

Valeriia-2 committed:
Стилизация страниц, добавление изображений, сетки, переменные, шрифты,

Valeriia-2 committed:
Контейнеры, лайки для страницы каталога, доп блоки

Valeriia-2 committed:
Пытаюсь исправить марджины

Valeriia-2 committed:
Поправила отступ

Valeriia-2 committed:
Удалила ссылки со шрифтами

Valeriia-2 committed:
Поменяла section на div

Valeriia-2 committed:
Подключила global, добавила box-sizing: border-box

feralist21 reviewed (changes requested):
Сборка запускается, с препроцессором все в порядке, ты умница :)
Давай подправим код в тех местах которые мы обсуждали на консультации и закроем эту задачу :)
Речь о правках шапки, контейнера, фоновых картинках.

Valeriia-2 committed:
Шапка (пока без открытия-закрытия)

Valeriia-2 committed:
Отступ

Valeriia-2 committed:
Поправила съехавший футер

Valeriia-2 committed:
Меню стало открываться и закрываться... аллилуя

Valeriia-2 committed:
Поправила таблицу

Valeriia-2 committed:
Поменяла десктопную версию на 940. Посмотрела в макете

Корректировать и убирать скролл потом буду

Valeriia-2 committed:
Удалила старый js код

Valeriia-2 committed:
Крестик! Открытие, закрытие меню

Valeriia-2 committed:
Повесила правильный бэкграунд на header top

Valeriia-2 committed:
Поправила иконки соц сетей

Valeriia-2 committed:
Единооразие кода стилей

Valeriia-2 committed:
Бэкграунд для открытия-закрытия меню для внутренних страниц

Valeriia-2 committed:
Поправки с отступами

Valeriia-2 committed:
Внутр страницы: сделала, чтобы открывающее меню не сдвигала контент

Valeriia-2 committed:
Добавила лого на внутр страницы, сделала поправки

Поправки:
для мобильной и планшетной версии, раньше весь. header top был кликабелным и переходилна главную страницу. Это поправила

Valeriia-2 committed:
Правки для лого

Valeriia-2 committed:
Добавила телефона(бэкграунды) на главную стр

Valeriia-2 committed:
Пометки валидатора

Valeriia-2 committed:
Поправила toggles

Valeriia-2 committed:
Добавила картинку quotes

Valeriia-2 committed:
Бэкграунды, лайки, контейнер каталога фотографий, единообразие кода поправки

feralist21 reviewed (approved):
Очень много работы было проделано. Я вижу как ты стараешься, ты молодец!)
